apps/.storybook/head.html

```diff
@@ -2,4 +2,5 @@
 <link rel="stylesheet" media="all" href="css/common.css">
 <link rel="stylesheet" media="all" href="css/applab.css">
 <link rel="stylesheet" media="all" href="css/droplet/droplet.min.css">
+<script src="https://use.fontawesome.com/8c9a5ecdc4.js"></script>
```

Comment:
Do we not have a local resource we can use (since this is not offline-friendly)?